serve.js

```diff
@@ -141,6 +143,13 @@ app.use('/fonts/material-icons/iconfont', express.static(
 // Endpoint to expose version number to client
 app.get('/version', (req, res) => res.send(version));
 
+
+
+app.get('/updates', async (req, res) => {
+    const changelog = await fs.promises.readFile(__dirname + '/CHANGELOG.md');
+    res.send(marked("" + changelog));
```

Comment:
Instead of using type coercion (adding a string to `changelog` to convert it from a `Buffer` to a string), use [`Buffer.prototype.toString`](https://nodejs.org/api/buffer.html#buffer_buf_tostring_encoding_start_end):
```suggestion
    res.send(marked(changelog.toString()));
```